specification/2.0/README.md

```diff
@@ -2125,7 +2125,7 @@ The length of the bufferView in bytes.
 
 #### bufferView.byteStride
 
-The stride, in bytes, between vertex attributes.  When this is not defined, data is tightly packed. When two or more accessors use the same bufferView, this field must be defined.
+The stride, in bytes, between vertex attributes.  When this is not defined, data is tightly packed. When two or more vertex data accessors use the same bufferView, this field must be defined.
```

Comment:
This reference section was auto-generated from the schema files.  Please make the same change in both places.

(Soon we may adopt an automated process to generate this reference from the schema).

Reply:
Specifically this line:

https://github.com/KhronosGroup/glTF/blob/6a3d023e63f3af893a3f66b7d8dc09f3c74ad21f/specification/2.0/schema/bufferView.schema.json#L29

Reply:
Thanks! Fixed.